config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
+# command to execute in running container
+# passed as argument to exec
+fndmnt = findmnt -n %(tmpfs_path)s 
+# Test will find container by CID instead of name, CID must appear
+# on stdout.
+run_options_csv = --detach=true,--tmpfs %(tmpfs_path)s
+tmpfs_path = /run
```

Comment:
Actually, IIRC python's ``ConfigParser`` operates only in line-forward mode, for interpreting those.  So unless I'm wrong, you _have_ to define it earlier in the file before you use it.